dist/breadcrumbs/breadcrumbs.css

```diff
@@ -1,13 +1,11 @@
 nav.breadcrumbs {
-  color: var(--breadcrumbs-item-foreground-color, var(--color-foreground-primary));
+  color: var(--breadcrumbs-item-foreground-color, var(--color-foreground-secondary));
   font-size: 0.75rem;
   height: -webkit-fit-content;
   height: fit-content;
   margin: 8px 0;
   min-height: 16px;
-  overflow-x: scroll;
```

Comment:
I think this would be the only concern. 
We have an old view where we are shrinking the breadcrumbs and showing the last item. 
This could be a breaking change in skin users. 

We might want to add a `breadcrumb--overflow` option to remove that overflow. 
Also @ianmcburnie thoughts if this is breaking or not?
For ebayui we would always enable that breadcrumb overflow since we can support the new designs out of the box.

Reply:
Yea, I can see how the older instances might structurally break with this change. Ok, I'll put the original styles back, add the modifier, and use it to cascade down to remove them for this updated version. I'm not a big fan of leaving deprecated styles in, but it's probably the cleanest way to go for now. 

I'm going to leave out the old implementation from the docs and we should add a note in the release notes that the older scrollable version will be deprecated with the next major.

Reply:
Yes agreed, let's remove the old documentation. We can have the `breadcrumb--overflow` class as default implementation. We should only see the new way of doing it. 
Put a deprecated comment also in the code so we can quickly find and remove it next major